janitor/functions/_numba.py

```diff
@@ -166,7 +166,7 @@ def _numba_equi_join(
     # this also lets us know if there are equi matches
     keep_rows = slice_starts < slice_ends
     if not keep_rows.any():
-        return None
+        return None, None
```

Comment:
@samukweku do we have documentation in-line about what the tuple elements `(None, None)` semantically mean?

Reply:
thanks for the prompt. I'll refactor the code to a clearer form